data/hooks/conf_regen/15-nginx

```diff
@@ -54,6 +54,14 @@ do_pre_regen() {
     [[ $main_domain != $domain ]] \
       && touch "${domain_conf_dir}/yunohost_local.conf" \
       || cp yunohost_local.conf "${domain_conf_dir}/yunohost_local.conf"
+    
+    domain_ca_type=$(yunohost domain cert-status ${domain} | grep CA_type | tr -d " ")
```

Comment:
Now that we have [jq](https://stedolan.github.io/jq/tutorial/) provided with YunoHost, you can do way more cleaner and strong code like this:

```
root@yunorunner:/home/vagrant# yunohost domain cert-status yunorunner.local --json | jq '.certificates."yunorunner.local".CA_type'
"Self-signed"
```